lib/rules/length-zero-no-unit/index.js

```diff
@@ -20,6 +20,13 @@ const messages = ruleMessages(ruleName, {
 	rejected: 'Unexpected unit',
 });
 
+const gridPropertiesToIgnore = [
+	'grid-template-columns',
+	'grid-template-rows',
+	'grid-auto-columns',
+	'grid-auto-rows',
+];
```

Comment:
What about move this in https://github.com/stylelint/stylelint/tree/master/lib/reference?

Reply:
Now I think this array can be removed at all. Since ```fr``` unit only works with grid family properties (correct me if it's not so) this check is useless. Do you agree?

Reply:
@fanich37 it makes sense! :)

Reply:
@hudochenkov @evilebottnawi removed.